packages/loading-indicator/src/BlobLoader/BlobLoader.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Player } from '@lottiefiles/react-lottie-player';
+import PropTypes from 'prop-types';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { Body, useUpdatedBaseFontSize } from '@leafygreen-ui/typography';
+
+import { descriptionThemeColor } from '../LoadingIndicator.styles';
+
+import animationJson from './animation';
+import { blobStyles, rootStyles } from './BlobLoader.styles';
+import { BlobLoaderProps } from './BlobLoader.types';
+
+/**
+ * Displays an animation of various brand shapes morphing from one to another paired with description text
+ */
+const BlobLoader = ({
+  baseFontSize: baseFontSizeProp,
+  description,
+  darkMode: darkModeProp,
+  className,
+  ...rest
+}: BlobLoaderProps) => {
+  const { theme } = useDarkMode(darkModeProp);
+  const baseFontSize = useUpdatedBaseFontSize(baseFontSizeProp);
+  return (
+    <div className={cx(rootStyles, className)} {...rest}>
+      <Player autoplay loop src={animationJson} style={blobStyles} />
+      {description && (
+        <Body color={descriptionThemeColor[theme]} baseFontSize={baseFontSize}>
```

Comment:
TIL we have a color prop on body

Reply:
What does it do? The HTML shows `<p class="leafygreen-ui-1tb6tuo" color="#5C6C75">Loading...</p>`

Reply:
My mistake! Replacing.